sale_automatic_workflow_payment_mode/data/automatic_workflow_data.xml

```diff
@@ -7,8 +7,10 @@
 <odoo noupdate="1">
     <record id="automatic_workflow_payment_filter" model="ir.filters">
         <field name="name">Automatic Workflow Payment Filter</field>
-        <field name="model_id">account.invoice</field>
-        <field name="domain">[('state', '=', 'open')]</field>
+        <field name="model_id">account.move</field>
```

Comment:
This field is a many2one:

```suggestion
<field name="model_id" ref="account.model_account_move" />
```

Reply:
I am checking v13 and v14 in runbot and the `model_id` field in `ir.filters` model is a selection field.

Reply:
Oh, sorry, I have mixed it with `ir.rule`.  The name of the field doesn't also help too much.